From 0e12f994c9e096087020d8ae2bd0d9f646839328 Mon Sep 17 00:00:00 2001
From: Devajith Valaparambil Sreeramaswamy
 <devajith.valaparambil.sreeramaswamy@cern.ch>
Date: Thu, 31 Oct 2024 09:32:20 +0100
Subject: [PATCH 39/52] Revert "Use raw string literals to correctly stringify
 '"' and '\\' in attributes."

This reverts commit 9660361a84f99b6f039c9fc7622445a421ac7db4.
---
 .../llvm-project/clang/utils/TableGen/ClangAttrEmitter.cpp | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/interpreter/llvm-project/clang/utils/TableGen/ClangAttrEmitter.cpp b/interpreter/llvm-project/clang/utils/TableGen/ClangAttrEmitter.cpp
index fbf61b0c87..89b88e386f 100644
--- a/interpreter/llvm-project/clang/utils/TableGen/ClangAttrEmitter.cpp
+++ b/interpreter/llvm-project/clang/utils/TableGen/ClangAttrEmitter.cpp
@@ -481,13 +481,12 @@ namespace {
     }
 
     void writeValue(raw_ostream &OS) const override {
-      OS << "R\\\"ATTRDUMP(\" << get" << getUpperName()
-         << "() << \")ATTRDUMP\\\"";
+      OS << "\\\"\" << get" << getUpperName() << "() << \"\\\"";
     }
 
     void writeDump(raw_ostream &OS) const override {
-      OS << "    OS << \" R\\\"ATTRDUMP(\" << SA->get" << getUpperName()
-         << "() << \")ATTRDUMP\\\"\";\n";
+      OS << "    OS << \" \\\"\" << SA->get" << getUpperName()
+         << "() << \"\\\"\";\n";
     }
   };
 
-- 
2.43.5

